pbsmetrics/metrics.go

```diff
@@ -61,6 +61,8 @@ const (
 	DemandUnknown DemandSource = "unknown"
 )
 
+const PublisherUnknown = "unknown"
```

Comment:
This is just a minor thing but I'd suggest moving this too either before the demand source related definitions on L56 or after on L73. This is just to keep related definitions together so it's easy to read.

Reply:
Done